<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabSynchronisation</class>
 <widget class="QWidget" name="tabSynchronisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	color: rgb(86, 184, 139);
	background-color: rgb(54, 54, 54);
	border-color: rgb(86, 184, 139);
	selection-background-color: rgb(58, 152, 112);
}

QTabWidget
{
background-color: rgb(85, 87, 83);
}

QTableWidget
{
background-color: rgb(85, 87, 83);
selection-background-color: rgb(58, 152, 112);
border-color: rgb(86, 184, 139);
}

QLabel#errorLabel
{
color: rgb(217, 174, 23);
}

QLineEdit#inputPath
{
background-color: rgb(85, 87, 83);
border-color: rgb(86, 184, 139);
}



QPushButton#dataDeleteButton
{
background-color: rgb(164, 0, 0);
color: rgb(46, 52, 54);
}

</string>
  </property>
  <widget class="QPushButton" name="create_configuration_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_configuration_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[enabled=true]{
color: rgb(86, 184, 139);
}

QPushButton[enabled=false]{
color: rgb(117, 138, 122);
}</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_configuration_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[enabled=true]{
color: rgb(86, 184, 139);
}

QPushButton[enabled=false]{
color: rgb(117, 138, 122);
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTableView" name="event_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>641</width>
     <height>251</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="force_trigger_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[enabled=true]{
color: rgb(86, 184, 139);
}

QPushButton[enabled=false]{
color: rgb(117, 138, 122);
}</string>
   </property>
   <property name="text">
    <string>force upload</string>
   </property>
  </widget>
  <widget class="QTableView" name="configuration_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>639</width>
     <height>279</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="event_view_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Synchronisation events</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
